packages/terra-action-footer/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Action Footer
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-action-footer.svg)](https://www.npmjs.org/package/terra-action-footer)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+{insert description}
```

Comment:
Need to add a description here, I'd recommend combining the one from `packages/terra-action-footer/docs/README.md` and `packages/terra-action-footer/docs/CenteredActionFooter.md` into a shortened version.

Reply:
Ah.. missed this one. Good catch! 👍